src/app.ts

```diff
@@ -23,34 +24,54 @@ app.get('/', (_: express.Request, res: express.Response) => {
 app.get('/flights', (req, res) => {
   const { query } = req;
   const seed = DateTime.fromISO(query.date).toISODate();
+  if (!seed) {
+    const dateFormatText = 'YYYY-MM-DD';
+    if (!query.date) {
```

Comment:
Can we move this check above the `DateTime.fromISO` line so that it does an early exit

Reply:
Ah good catch!